Formula/austin.rb

```diff
@@ -3,7 +3,7 @@ class Austin < Formula
   homepage "https://github.com/P403n1x87/austin"
   url "https://github.com/P403n1x87/austin/archive/v1.0.1.tar.gz"
   sha256 "f86804f777f2f466ddede5d530d3ca67582b2a1467d000662d81272d6e9c5639"
-
+  license "GPL-3.0"
```

Comment:
Is this a mistake?

Reply:
Looks like GPLv3: https://github.com/P403n1x87/austin/blob/master/LICENSE.md

Reply:
Sorry, I meant the blank line...

I only bring it up in case `brew style` doesn't like it (I can't remember if it will complain). Otherwise, it's probably bikeshedding on my part...

Reply:
Brew style doesn't complain about this so it's not necessary to change it.